### PR TITLE
Pci stages

### PR DESCRIPTION
**Pull request by jinatdatadog**
### What does this PR do? What is the motivation?
Updating the onboarding documentation for PCI customers

### Merge instructions

- [ X ] Please merge after reviewing

### CODE REVIEW

File: layouts/shortcodes/pci-apm.md [hunk 1]
```diff
@@ -1,15 +1,32 @@
-1. Contact [Datadog support][101] or your [Customer Success Manager][102] to request that the org be configured as a PCI-compliant org and discuss the necessary paperwork to complete the PCI requirements.
-1. If not already enabled, [Audit Trail][103] is automatically enabled when the org is configured as PCI-compliant. Audit Trail *must* be enabled and remain enabled for PCI DSS compliance.
-1. After Datadog support or Customer Success confirms that the org is PCI DSS compliant, configure the respective configuration file to send spans to the dedicated PCI-compliant endpoint:
+To set up PCI compliant Application Performance Monitoring, you must meet the following requirements:
+- If not already enabled, Audit Trail is automatically enabled when the org is configured as PCI compliant. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
+- Your Datadog organization is in the US1 site.
+- All spans sent to the PCI endpoints using HTTPS only. If you are using the Agent, you should enforce HTTPS transport.
+- **All** your endpoints will need to be changed to the PCI endpoints.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-apm.md:5]:
```suggestion
- **All** your endpoints need to be changed to the PCI endpoints. 
```

File: layouts/shortcodes/pci-apm.md [hunk 2]
```diff
@@ -1,15 +1,32 @@
-1. Contact [Datadog support][101] or your [Customer Success Manager][102] to request that the org be configured as a PCI-compliant org and discuss the necessary paperwork to complete the PCI requirements.
-1. If not already enabled, [Audit Trail][103] is automatically enabled when the org is configured as PCI-compliant. Audit Trail *must* be enabled and remain enabled for PCI DSS compliance.
-1. After Datadog support or Customer Success confirms that the org is PCI DSS compliant, configure the respective configuration file to send spans to the dedicated PCI-compliant endpoint:
+To set up PCI compliant Application Performance Monitoring, you must meet the following requirements:
+- If not already enabled, Audit Trail is automatically enabled when the org is configured as PCI compliant. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
+- Your Datadog organization is in the US1 site.
+- All spans sent to the PCI endpoints using HTTPS only. If you are using the Agent, you should enforce HTTPS transport.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-apm.md:4]:
```suggestion
- All spans sent to the PCI endpoints using HTTPS only. If you are using the Agent to send spans, you should enforce HTTPS transport.
```

File: layouts/shortcodes/pci-apm.md [hunk 3]
```diff
@@ -1,15 +1,32 @@
...
+3. For example, add the following lines to the Agent configuration file:
 ```
 apm_config:
   apm_dd_url: <https://trace-pci.agent.datadoghq.com>
 ```
+4. All spans that are sent to the PCI compliant endpoint(s) automatically have a set of [Sensitive Data Scanner][106] PCI rules that are applied to scrub any cardholder data. These dedicated PCI rules must be enalbed for PCI DSS compliance and are included with no additional charge.
+
+To finish onboarding and be moved to **compliant**:
+1. Inform your [Datadog support][101] or your [Customer Success Manager][102] that you have moved over **all** your endpoints to the PCI compliant endpoint(s).
+2. Once confirmed by Datadog, your spans will be configured to be PCI compliant. Only now is your Application Performance Monitoring considered PCI compliant.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-apm.md:23]:
```suggestion
2. Once confirmed by Datadog, your span configuration and Application Performance Monitoring is considered PCI-compliant.
```

File: layouts/shortcodes/pci-apm.md [hunk 4]
```diff
@@ -1,15 +1,32 @@
...
+
+If you have any questions about how your now PCI compliant Application Performance Monitoring satisfies the applicable requirements under PCI DSS, contact your account manager. For PCI compliant Log Management, see [here][104].
```
Comment (rtrieu) [inline layouts/shortcodes/pci-apm.md:25]:
```suggestion
If you have any questions about how your now PCI-compliant Application Performance Monitoring satisfies the applicable requirements under PCI DSS, contact your account manager. See information on setting up [PCI-compliant Log Management][104].
```

File: layouts/shortcodes/pci-logs.md [hunk 5]
```diff
@@ -1,22 +1,34 @@
-1. Contact [Datadog support][101] or your [Customer Success Manager][102] to request that the org be configured as a PCI-compliant org and discuss the necessary paperwork to complete the PCI requirements.
-1. If not already enabled, [Audit Trail][103] is automatically enabled when the org is configured as PCI-compliant. Audit Trail *must* be enabled and remain enabled for PCI DSS compliance.
-1. After Datadog support or Customer Success confirms that the org is PCI DSS compliant, configure the respective configuration file to send logs to the dedicated PCI-compliant endpoint:
+To set up PCI compliant Log Management, you must meet the following requirements:
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:1]:
```suggestion
To set up PCI-compliant Log Management, you must meet the following requirements:
```

File: layouts/shortcodes/pci-apm.md [hunk 6]
```diff
@@ -1,15 +1,32 @@
-1. Contact [Datadog support][101] or your [Customer Success Manager][102] to request that the org be configured as a PCI-compliant org and discuss the necessary paperwork to complete the PCI requirements.
-1. If not already enabled, [Audit Trail][103] is automatically enabled when the org is configured as PCI-compliant. Audit Trail *must* be enabled and remain enabled for PCI DSS compliance.
-1. After Datadog support or Customer Success confirms that the org is PCI DSS compliant, configure the respective configuration file to send spans to the dedicated PCI-compliant endpoint:
+To set up PCI compliant Application Performance Monitoring, you must meet the following requirements:
+- If not already enabled, Audit Trail is automatically enabled when the org is configured as PCI compliant. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-apm.md:2]:
```suggestion
- Audit Trail must be enabled and remain enabled for PCI DSS compliance. If you haven't already enabled Audit Trail, it is _automatically enabled_ once the org is configured as PCI-compliant (after following the steps below).
```

File: layouts/shortcodes/pci-logs.md [hunk 7]
```diff
@@ -1,22 +1,34 @@
...
+- If not already enabled, Audit Trail is automatically enabled when the org is configured as PCI compliant. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:2]:
```suggestion
- Audit Trail must be enabled and remain enabled for PCI DSS compliance. If you haven't already enabled Audit Trail, it is _automatically enabled_ once the org is configured as PCI-compliant (after following the steps below).
```

File: layouts/shortcodes/pci-logs.md [hunk 8]
```diff
@@ -1,22 +1,34 @@
...
+- Your Datadog organization is in the US1 site.
+- All logs sent to the PCI endpoints using HTTPS only. If you are using the Agent, you should enforce HTTPS transport.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:4]:
```suggestion
- All logs sent to the PCI endpoints using HTTPS only. If you are using the Agent to send logs, you should enforce HTTPS transport.
```

File: layouts/shortcodes/pci-logs.md [hunk 9]
```diff
@@ -1,22 +1,34 @@
...
+- **All** your endpoints will need to be changed to the PCI endpoints.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:5]:
```suggestion
- **All** your endpoints need to be changed to the PCI endpoints. 
```

File: layouts/shortcodes/pci-logs.md [hunk 10]
```diff
@@ -1,22 +1,34 @@
...
-1. If not already enabled, [Audit Trail][103] is automatically enabled when the org is configured as PCI-compliant. Audit Trail *must* be enabled and remain enabled for PCI DSS compliance.
-1. After Datadog support or Customer Success confirms that the org is PCI DSS compliant, configure the respective configuration file to send logs to the dedicated PCI-compliant endpoint:
+To set up PCI compliant Log Management, you must meet the following requirements:
+- If not already enabled, Audit Trail is automatically enabled when the org is configured as PCI compliant. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
+- Your Datadog organization is in the US1 site.
+- All logs sent to the PCI endpoints using HTTPS only. If you are using the Agent, you should enforce HTTPS transport.
+- **All** your endpoints will need to be changed to the PCI endpoints. 
+- You may request access to the PCI Attestation of Compliance and Customer Responsibility Matrix on [Datadog's Trust Center][105] - note that these documents are only applicable once you have finished all the onboarding steps and have been manually configured to compliant by Datadog support.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:6]:
```suggestion
- You may request access to the PCI Attestation of Compliance and Customer Responsibility Matrix on [Datadog's Trust Center][105] - note that these documents are only applicable once you have finished all the onboarding steps and have been manually configured to be compliant by Datadog support.
```

File: layouts/shortcodes/pci-apm.md [hunk 11]
```diff
@@ -1,15 +1,32 @@
...
+- Your Datadog organization is in the US1 site.
+- All spans sent to the PCI endpoints using HTTPS only. If you are using the Agent, you should enforce HTTPS transport.
+- **All** your endpoints will need to be changed to the PCI endpoints. 
+- You may request access to the PCI Attestation of Compliance and Customer Responsibility Matrix on [Datadog's Trust Center][105] - note that these documents are only applicable once you have finished all the onboarding steps and have been manually configured to compliant by Datadog support.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-apm.md:6]:
```suggestion
- You may request access to the PCI Attestation of Compliance and Customer Responsibility Matrix on [Datadog's Trust Center][105] - note that these documents are only applicable once you have finished all the onboarding steps and have been manually configured to be compliant by Datadog support.
```

File: layouts/shortcodes/pci-logs.md [hunk 12]
```diff
@@ -1,22 +1,34 @@
...
 ```
 logs_config:
   logs_dd_url: <agent-http-intake-pci.logs.datadoghq.com:443>
 ```
+4. All logs that are sent to the PCI compliant endpoint(s) automatically have a set of [Sensitive Data Scanner][106] PCI rules that are applied to scrub any cardholder data. These dedicated PCI rules must be enalbed for PCI DSS compliance and are included with no additional charge.
 
-**Note**: The port must be included in the configuration. PCI compliance uses HTTPS log forwarding only. If you are using the Agent, you should [enforce HTTPS transport][104].
+To finish onboarding and be moved to **compliant**:
+1. Inform your [Datadog support][101] or your [Customer Success Manager][102] that you have moved over **all** your endpoints to the PCI compliant endpoint(s).
+2. Once confirmed by Datadog, your Logs will be configured to be PCI compliant. Only now is your Log Management considered PCI compliant.
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:25]:
```suggestion
2. Once confirmed by Datadog, your Logs and Log Management is considered to be PCI-compliant.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some feedback for your review!

File: layouts/shortcodes/pci-logs.md [hunk 13]
```diff
@@ -1,22 +1,34 @@
...
 ```
+4. All logs that are sent to the PCI compliant endpoint(s) automatically have a set of [Sensitive Data Scanner][106] PCI rules that are applied to scrub any cardholder data. These dedicated PCI rules must be enalbed for PCI DSS compliance and are included with no additional charge.
 
-**Note**: The port must be included in the configuration. PCI compliance uses HTTPS log forwarding only. If you are using the Agent, you should [enforce HTTPS transport][104].
+To finish onboarding and be moved to **compliant**:
+1. Inform your [Datadog support][101] or your [Customer Success Manager][102] that you have moved over **all** your endpoints to the PCI compliant endpoint(s).
+2. Once confirmed by Datadog, your Logs and Log Management is considered to be PCI-compliant.
 
-If you have any questions about how the Log Management service satisfies the applicable requirements under PCI DSS, contact your account manager.
+If you have any questions about how your now PCI compliant Log Management satisfies the applicable requirements under PCI DSS, contact your account manager. For PCI compliant Application Performance Monitoring, see [here][104].
```
Comment (rtrieu) [inline layouts/shortcodes/pci-logs.md:27]:
```suggestion
If you have any questions about how your now PCI compliant Log Management satisfies the applicable requirements under PCI DSS, contact your account manager. See information on setting up [PCI-compliant Log Management][104].
```

Comment (rtrieu) [reply layouts/shortcodes/pci-logs.md:27]: Should this actually be pointing to `/data_security/pci_compliance/?tab=logmanagement` instead?

**Review by rtrieu — CHANGES_REQUESTED** [review]
A few more things!

File: layouts/shortcodes/pci-logs.md (hunk 13 above)
Comment (jinatdatadog) [reply layouts/shortcodes/pci-logs.md:27]: Nope - I want to point and refer to APM on this Logs tab and refer to Logs on the APM tab - if this makes sense. So it should still be the original " For PCI compliant Application Performance Monitoring, see [here]", here pointing to the APM page

Comment (rtrieu) [reply layouts/shortcodes/pci-logs.md:27]: Updated my suggestion!

**Review by rtrieu — APPROVED** [review]
LGTM!

**Pull request merged**